_includes/docs-info.html

```diff
@@ -42,17 +42,19 @@ <h6>For Users</h6>
         <a name="tutorials"></a>
         <h5>Tutorials for the finer steps</h5>
             <div class="row docs-details">
-                <a href="https://bit.ly/2Fyn97v">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated July 21st 2017.
+                <a href="https://bit.ly/3q73CTo">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated September 2022.
```

Comment:
https://docs.google.com/presentation/d/1ycQyH7THwjlwe7PTaEwhKAOt0FbFrsrfo93Zn1wcc3E/present?slide=id.p18

Sentence is just "SPHARM-PDM will be ready to use when"

Reply:
The sentence fragment appears to affect multiple tutorials

Reply:
The fragments have been updated.